test/common_quantization.py

```diff
@@ -123,6 +123,15 @@ def checkScriptable(self, orig_mod, calib_data, check_save_load=False):
         scripted = torch.jit.script(orig_mod)
         self._checkScriptable(orig_mod, scripted, calib_data, check_save_load)
 
+        # Use first calib_data entry as trace input
+        #
+        # TODO: Trace checking is blocked on this issue:
```

Comment:
Create an issue, and reference this there.

Reply:
There's already an issue listed right below... No need to create a duplicate and clog up the issue tracker